base/acme/webapps/acme/index.jsp

```diff
@@ -0,0 +1,139 @@
+<!--
+Copyright Red Hat, Inc.
+
+SPDX-License-Identifier: GPL-2.0-or-later
+-->
+<html>
+<head>
+    <title>ACME Responder</title>
+    <link rel="stylesheet" href="css/patternfly-4.35.2.css">
+    <script src="js/jquery-3.5.1.js"></script>
+
+    <script>
+$(function() {
+
+    // replace ACME_URL with actual ACME URL
+    var i = window.location.href.lastIndexOf('/');
+    var acme_url = window.location.href.substring(0, i);
+
+    $("pre").each(function() {
+        var content = this.innerHTML;
+        this.innerHTML = content.replace("ACME_URL", acme_url);
```

Comment:
Could we do `innerText` instead? :-) I think that'd be a _little_ safer. My 2c.

Reply:
OK, let me try that.

Reply:
Updated.